_rules/focusable-no-keyboard-trap-standard-nav-a1b64e.md

```diff
@@ -42,6 +42,8 @@ There are no major accessibility support issues known for this rule.
 
 ## Background
 
+[Standard keyboard navigation](#standard-keyboard-navigation) can move focus both forward and backward. Therefore, cycling back to the browser UI can be done both by moving forward through the tab order and by moving backwards. It is not fully clear whether [Success Criterion 2.1.2 No keyboard trap][sc212] requires navigation in both directions. Therefore, to stay on the safe side, this rule does not constrain navigation direction.
```

Comment:
This is an assumption. I think we put it there. Also, might be good to open an issue with AGWG to just ask this question.

My recollection of this though was that we'd said that forward navigation was required, but backward navigation was not. Am I misremembering that?

Reply:
> My recollection of this though was that we'd said that forward navigation was required, but backward navigation was not. Am I misremembering that?

As per [CG discussion of July 14th](https://www.w3.org/2022/07/14-act-r-minutes.html#item05) and the following [summary of action](https://github.com/act-rules/act-rules.github.io/issues/1885#issuecomment-1184535392), we decided that either forward or backward was enough (to stay on the safe side).

I agree this should be an Asumption 👍

Reply:
> This is an assumption. I think we put it there. Also, might be good to open an issue with AGWG to just ask this question.

https://github.com/w3c/wcag/issues/2678

Putting this on hold, waiting to see if we have an answer.